tests/Php56/Php56Test.php

```diff
@@ -80,7 +80,7 @@ public function testLdapEscape($subject, $ignore, $flags, $expected)
     public function provideLdapEscapeValues()
     {
         return array(
-            array('foo=bar(baz)*', null, p::LDAP_ESCAPE_DN, 'foo\3dbar(baz)*'),
```

Comment:
I added to it to include the leading/trailing spaces and a carriage return, as that was the test that seemed to be dealing strictly with escaping a DN. However, if you want, I can leave that test alone and include a completely separate line for testing a DN with leading/trailing spaces and a carriage return.